cl_tf2/src/cl-transforms-plugin/package.lisp

```diff
@@ -36,4 +36,4 @@
    ;; point-stamped
    point-stamped make-point-stamped copy-point-stamped point
    ;; pose-stamped
-   pose-stamped make-pose-stamped copy-pose-stamped pose))
```

Comment:
As discussed earlier today, I'd like to have this explicitedly exported in the ```package.lisp```. Like this, there is one place where all the exported API can be found by people.